### DISCUSSION TITLE
Create util to handle padding

### DISCUSSION THREAD

**Discussion by bukinoshita** [resend/react-email#587] [Ideas]
Whenever the user passes a style prop with padding, we should have an util to handle the padding by adding the `msoPaddingAlt` attribute as well.

const padding = (style: React.CSSProperties) => {
  // Check if there's any padding
  return { ...style, msoPaddingAlt: style.padding }
}

**Comment by mohitmohlia** [comment]
i will take this enhancement

**Comment by mohitmohlia** [comment]
i am unable to setup the project, `yarn install` is stuck on forever.

<img width="440" alt="Screenshot 2023-03-04 at 5 12 39 PM" src="https://user-images.githubusercontent.com/46368810/222898339-7dbcbad4-03e4-43f5-801d-73f2b4fd8b72.png">

In another terminal if you run `yarn install`

<img width="482" alt="Screenshot 2023-03-04 at 5 14 25 PM" src="https://user-images.githubusercontent.com/46368810/222898376-7006d21c-e4e7-48b5-8cc3-ea7a8e118d69.png">

running `yarn dev` fails 

<img width="767" alt="Screenshot 2023-03-04 at 5 17 37 PM" src="https://user-images.githubusercontent.com/46368810/222898503-808d8d29-ceb6-4b16-a617-ea08379bbcd5.png">

looks like dependencies were installed incorrectly for docs

**Comment by bukinoshita** [comment]
Can you try with latest, please?
